### DISCUSSION TITLE
角色培养帮帮！！

### DISCUSSION THREAD

**Discussion by LeonixBeria** [share121/inter-knot#1187] [其他]
![3e3793b7839c3a1e4c18457985a73488](https://github.com/user-attachments/assets/57a3acf3-10f7-4b2c-98f8-90a4c72af3a9)
0+1的鲨鱼妹捏，现在是四冰套配二暴击套，最近在想暴击是否有点溢出了，要不要换成二穿透套

**Comment by themagicmoon** [comment]
不道啊，去问问神奇海螺吧

**Comment by int-maxxx** [comment]
这个委托我接了，暴击衣换爆伤衣
